packages/devtools_app/lib/src/debugger/evaluate.dart

```diff
@@ -172,8 +172,10 @@ class _ExpressionEvalFieldState extends State<ExpressionEvalField> {
   }
 
   Future<String> _getBestStringValue(InstanceRef response) async {
+    // Return the string value if present and not truncated.
+    // Note: string value is truncated iff not null and true
```

Comment:
typo: iff

"not null and true" looks a little ambiguous - "not null and not true" or "neither null or true" may be clearer.

Reply:
I actually mean '(value != null) && (value == true)'. not sure how to express it the best:) 'Not null and not false' maybe, if it is not too convoluted?

Reply:
Oh sorry, I misunderstood the comment thinking it was referring to the return value inside the `if` but it's describing when the string in the `InstanceRef` is truncated.

In which case, I think

> [valueAsStringIsTruncated] is not null and is set to true

may be intended to say

> [valueAsString] is not null and [valueAsStringIsTruncated] is true

Reply:
no i actually mean [valueAsStringIsTruncated] is not null and [valueAsStringIsTruncated] is true.

Reply:
I actually meant the [valueAsStringIsTruncated] in both cases. I changes it to following to remove the confusion:

    // Note: [valueAsStringIsTruncated] is actually a tri-state: 
    // string value is not truncated iff [valueAsStringIsTruncated]
    // is null or [valueAsStringIsTruncated] is false.